src/connection-manager/index.ts

```diff
@@ -132,6 +134,24 @@ export interface ConnectionManagerInit {
    * tagged with KEEP_ALIVE up to this timeout in ms. (default: 60000)
    */
   startupReconnectTimeout?: number
+
+  /**
+   * A list of multiaddrs that will always be allowed to open connections to
```

Comment:
```suggestion
   * A list of multiaddrs that will always be allowed (except if they are in deny list) to open connections to
```
or some other wording indicating that deny list has higher priority